backend/internal/handlers/venues/venues.go

```diff
@@ -0,0 +1,40 @@
+package venues
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/google/uuid"
+)
+
+func (s *Service) DeleteVenue(c *fiber.Ctx) error {
+	fmt.Println("Pinging correct service")
+
+	venueID := c.Params("venueId")
```

Comment:
you can simplify this block by deferring the length check to [`uuid.Parse`](https://github.com/google/uuid/blob/v1.6.0/uuid.go#L68)